src/VisualStudio/LiveShare/Impl/Shims/TypeScriptHandlerShims.cs

```diff
@@ -147,7 +147,7 @@ protected override async Task FindImplementationsAsync(IFindUsagesService findUs
     }
 
     [ExportLspRequestHandler(LiveShareConstants.TypeScriptContractName, Methods.TextDocumentSignatureHelpName)]
-    internal class TypeScriptSignatureHelpHandlerShim : SignatureHelpHandler, ILspRequestHandler<TextDocumentPositionParams, SignatureHelp, Solution>
+    internal class TypeScriptSignatureHelpHandlerShim : SignatureHelpHandler, ILspRequestHandler<TextDocumentPositionParams, SignatureHelp?, Solution>
```

Comment:
FYI @uniqueiniquity

Reply:
Oh yeah forgot about these.  Are these dead yet @uniqueiniquity ?  Should we just remove them

Reply:
Sorry, was out of the office for a few days.
Signature help definitely is, though I think others aren't (e.g. navbar stuff).